src/Cache_Command.php

```diff
@@ -339,4 +339,36 @@ public function type( $args, $assoc_args ) {
 		WP_CLI::line( $message );
 	}
 
+	/**
+	 * Removes all cache items in a group, if the object cache implementation supports it.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <group>
+	 * : Cache group key.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Clear cache group.
+	 *     $ wp cache clear-group my_group
+	 *
+	 * @subcommand clear-group
```

Comment:
@spacedmonkey Should it be `wp cache clear-group`, `wp cache delete-group`, or `wp cache flush-group`?

Reply:
@tillkruss Maybe you might have an opinion on naming ^ ?

Reply:
I'd stick with `wp cache flush-group` to be identical to `wp_cache_flush_group()`, but I'm a fan of `@alias wp cache clear-group`.

Reply:
Hm... I think it should've been named `wp_cache_delete_group()` in core 😁

`flush-group` is too close to `flush`, in my opinion. But, I guess we should be consistent with naming.

@wp-cli/committers Any other opinions to bring into the mix?

Reply:
Yeah, alias `delete` and `clear` makes sense.

Reply:
Having worked on it a bit more I think I would prefer `flush-group` it might be close to the existing `flush` but I think its more consistent that way.

Reply:
> Having worked on it a bit more I think I would prefer `flush-group` it might be close to the existing `flush` but I think its more consistent that way.

Sounds good to me 👍